src/printer.js

```diff
@@ -3585,14 +3581,16 @@ function printMemberChain(path, options, print) {
       break;
     }
   }
-  for (; i + 1 < printedNodes.length; ++i) {
-    if (
-      isMemberish(printedNodes[i].node) &&
-      isMemberish(printedNodes[i + 1].node)
-    ) {
-      currentGroup.push(printedNodes[i]);
-    } else {
-      break;
+  if (printedNodes[0].node.type !== "CallExpression") {
```

Comment:
If guard was to prevent `[CallExpression, MemberExpression, MemberExpression]` in the first group.

i.e.:

```js
expect(pet).to.have.property("OwnerAddress").that.deep.equals({
  AddressLine1: "Alexanderstrasse",
});
```

was being formatted as


```js
expect(pet).to.have
  .property("OwnerAddress")
  .that.deep.equals({
    AddressLine1: "Alexanderstrasse",
  });
```

Reply:
Wow, this was mentioned in https://github.com/prettier/prettier/issues/21#issuecomment-275202447